dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api_test.cc

```diff
@@ -206,7 +204,9 @@ void PrintDeviceInfo() {
 bool ShouldUseHwDecoder() {
   // HW decoder is disabled for drivers < 455.x, see
   // dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h for details
+  nvml::Init();
   static float driver_version = nvml::GetDriverVersion();
+  nvml::Shutdown();
   static bool device_supports_hw_decoder = nvml::isHWDecoderSupported();
   return device_supports_hw_decoder && driver_version >= 455;
```

Comment:
This is a potential perf hit.
```suggestion
  static const float driver_version = []() {
    nvml::Init();
    float ver = nvml::GetDriverVersion();
    nvml::Shutdown();
    return ver;
  };
  static const bool device_supports_hw_decoder = nvml::isHWDecoderSupported();
  return device_supports_hw_decoder && driver_version >= 455;
```

Reply:
In the hot path, I would go for something faster. This is just a test. I don't think it will affect it much.